expense-splitter/src/components/expense/CreateExpense.jsx

```diff
@@ -346,7 +350,15 @@ export default function CreateExpense() {
         <div className="mb-8">
           {watchedValues["group"] && (
             <>
-              <h2 className="mb-4">Weight Contributions:*</h2>
+              <h2 className="mb-4">
+                Weight Contributions:*
+                <Tooltip title={weightInfo} placement="right">
+                  <i
```

Comment:
In mobile view, the message is cut off. Try placement="bottom-end"

Reply:
I forgot to test mobile view XD

I can't trigger it on mobile view for some reason ...

Reply:
And of course I figured it out as soon as I wrote that